internal/app/agent/service.go

```diff
@@ -173,6 +180,31 @@ func (a *agent) Control(uuid, cmdStr string) error {
 	return nil
 }
 
+// Message for this command
+// "[{"bn":"1:", "n":"config", "vs":"export, /configs/export/config.toml, config_file_content"}]"
+// config_file_content is base64 encoded marshaled structure representing service conf
+// b, _ := toml.Marshal(cfg)
```

Comment:
Add something like `Example of creation:` before these lines so that they do not look like commented code, which we avoid.

We also always avoid ignoring errors, so this line `b, _ := toml.Marshal(cfg)` is giving a bad example.

Reply:
Also indent example lines to demark that they are not just commented code.